### DISCUSSION TITLE
how to get current/old sortingState

### DISCUSSION THREAD

**Discussion by 0xMALVEE** [TanStack/table#5196] [Q&A]
![image](https://github.com/TanStack/table/assets/35369563/e08bdfc9-c4a3-4457-801b-0d8caf5a54b9)
I want to get the sorting value that we are trying to change. using the onSortingChange method. problem is in functionalUpdate I need to pass the current sortingstate. how can I get the current sorting state.

**Comment by yamatohagi** [comment]
![スクリーンショット 2024-03-11 23 35 29](https://github.com/TanStack/table/assets/53859640/89116e61-7128-4dc7-bd1d-f284e37154b7)

Anything's okay
